### ISSUE TITLE
sites rejecting _ (underscore) query parameter

### ISSUE THREAD

**Issue by aronatkins** [samuelclay/NewsBlur#1877]
Some sites (e.g. https://www.jwz.org/blog/) have started returning errors when receiving unsupported query arguments, like `_`.

```bash
curl "https://cdn.jwz.org/blog/feed/?_=testing"
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
 <channel>
  <generator>jwz</generator>
  <title>jwz</title>
  <link>https://www.jwz.org/blog</link>
  <description></description>
  <language>en</language>
  <webMaster>jwz@jwz.org (jwz)</webMaster>
  <pubDate>Sat, 29 Jun 2024 18:14:51 GMT</pubDate>
  <lastBuildDate>Sat, 29 Jun 2024 18:14:51 GMT</lastBuildDate>
  <atom:link href="https://www.jwz.org/blog" rel="self" type="application/rss+xml" />
  <item>
   <link>https://www.jwz.org/blog/feed/?_=testing</link>
   <guid isPermaLink="false">ZoBPGw</guid>
   <title>Error</title>
   <description><![CDATA[unknown parameter "_" <BR><BR>

If you are seeing this, <BR>
your feed reader is badly behaved. <BR>
Use a different one.]]></description>
  </item>
 </channel>
</rss>
```

Background / motivation: https://www.jwz.org/blog/2024/06/blocking-bogus-url-parameters/

These errors have started showing up in my NewsBlur feed for this site.

Could you confirm that NewsBlur is added the `_` query argument? Could that behavior be removed or at least made configurable?

**Comment by aronatkins** [comment]
Likely related:
https://github.com/samuelclay/NewsBlur/blob/db30d559a08a3d7014668a3c29e765fd0560d4db/utils/feed_fetcher.py#L114-L115

**Comment by samuelclay** [comment]
This is fixed in 546f338b8c8ffebfebf16fda45a9c7c18de07f9d
